### PR TITLE
Preview fix lint

### PR DESCRIPTION
**Pull request by svFiserv**
view merge conflicts

### CODE REVIEW

**Comment by wwfiserv** [comment]
@svFiserv are you going to fix the merge conflicts or are you unsure and want me to review/fix them?

**Comment by svFiserv** [comment]
Hi,
I will fix them.

If I am stuck I will reach out to you.

Thank you,
Srividya
________________________________

File: docs/Resources/Master-Data/Balance-Components.md [hunk 1]
```diff
@@ -19,8 +19,7 @@ The below table identifies the parameters in the `balance` object.
 | `beginngingBalance` | *number* | 18,3 | Credit card number |
 | `endingBalance` | *number* | 18,3 | Account ending balance |
 | `currency` | *string* | 3 | ISO 3 Currency Format |
-| `blanceType` | *string* | 17 |  Indicates <a href="../docs?path=docs/Resources/Master-Data/Balance-Components.md#balance-type">type</a> of account balance represents. |
-
+| `blanceType` | *string* | 17 |  Indicates [the type](?path="../docs?path=docs/Resources/Master-Data/Balance-Components.md#balance-type") of account balance represents. |
```
Comment (wwfiserv) [inline docs/Resources/Master-Data/Balance-Components.md:22]: This should be an anchor link to #balance-type

Also for reference, to create a link in VS right click the document you want to link to and copy relative path, then paste it after ?path= example: ?path=docs/Resources/Master-Data/Balance-Components.md#balance-type

File: docs/Resources/Master-Data/Card-Details.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ The below table identifies the parameters in the `cardDetails` object.
 | `fastFundsCrossBorderNonMoneyTransfer` | *string* | 13 | Indicates if fast funds is supported for domestic non-money transfer OCTs. **Visa Only**. |
 | `fastFundsCrossBorderGambling` | *string* | 13 | Indicates if fast funds is supported for cross-border gambling OCTs. Visa Only. **Valid Values:** |
 | `prepaidIndicator` | *string* | 13 | Indicates whether the card is prepaid. **Valid Values:** *NON_PREPAID, PREPAID* |
-| `anonymousPrepaidIndicator` | *string* | 26 | Identifies whether Anonymous Prepaid is supported, like AMLD5 (Anti-Money Laundering Directive) compliant, non-AMLD5 compliant, or non-anonymous prepaid programs / not a prepaid program. Mastercard Only. **Valid Values:** _AMLD5_NON_COMPLIANT, _AMLD4_COMPLIANT_EQUIVALENT, *NON_ANONYMOUS* |
+| `anonymousPrepaidIndicator` | *string* | 26 | Identifies whether Anonymous Prepaid is supported, like AMLD5 (Anti-Money Laundering Directive) compliant, non-AMLD5 compliant, or non-anonymous prepaid programs / not a prepaid program. Mastercard Only. **Valid Values:** _AMLD5_NON_COMPLIANT, *AMLD4_COMPLIANT_EQUIVALENT*, *NON_ANONYMOUS* |
```
Comment (wwfiserv) [inline docs/Resources/Master-Data/Card-Details.md:51]: Verify all of these in this table, but they should be italic with valid values being bold and look like this ***Valid Values:** AMLD5_NON_COMPLIANT, AMLD4_COMPLIANT_EQUIVALENT, NON_ANONYMOUS*

File: docs/Resources/Master-Data/Debit-Pinless-Indicator.md [hunk 3]
```diff
@@ -2,7 +2,8 @@
 tags: [card-details, debit-network-pinless-details, master-data]
 ---
 
-# Debit Network Pinless Details 
-The Debit Network Pinless Details are returned in the [account information lookup](?path=docs/Resources/API-Documents/Payments_VAS/Information-Lookup.md) as part of the `cardDetails` object. 
+# Debit Network Pinless Details
```
Comment (wwfiserv) [inline docs/Resources/Master-Data/Debit-Pinless-Indicator.md:5]: I know this wasn't you, but for consistency can you update to PINless for all references

File: docs/Resources/Master-Data/Merchant-Category-Code.md [hunk 4]
```diff
@@ -9,7 +9,6 @@ Each business is assigned a Merchant Category Code (MCC), which is determined by
 <!-- theme: example -->
 > Example: All 'Inbound telemarketing merchants' will be assigned with MCC 5967 and all 'Hotels and Resorts' will have 7011.
 
-
 - **Static MCC** : The MCC is automatically set based on the merchant processing account. If incorrect, this would need to be updated by the account representative.
```
Comment (wwfiserv) [inline docs/Resources/Master-Data/Merchant-Category-Code.md:13]: Not sure why this wasn't caught but lists shouldn't have a break between them. Also formatting the colon : should also be bold.

File: docs/Resources/Master-Data/Payment-Receipt.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ The payment receipt also known as a Customer Transaction Record (CTR), is return
 
 ---
 
-## Payment Receipt
+**Payment receipt object:**
```
Comment (wwfiserv) [inline docs/Resources/Master-Data/Payment-Receipt.md:11]: Just remove this header, description and the prior horizontal line (line 9)

**Comment by svFiserv** [comment]
Hi William,

Have made the suggested changes.

docs/Archive/Payment-JS/JSV1-Request.md<https://urldefense.com/v3/__https://github.com/Fiserv/Commerce-Hub/pull/1386*discussion_r1829438760__;Iw!!P9vvK-4S!mlbvk1v5p2tGEMLgISVw-KEpOvsbc03VA_R_lCu7XkG_SkLZbDVUCSJm3EspkuAVZp5j1ckkvqC4Gr_aR5_VhDvozK5WDExtseTHduc$>:
Done

docs/Resources/Master-Data/Balance-Components.md<https://urldefense.com/v3/__https://github.com/Fiserv/Commerce-Hub/pull/1386*discussion_r1829545098__;Iw!!P9vvK-4S!mlbvk1v5p2tGEMLgISVw-KEpOvsbc03VA_R_lCu7XkG_SkLZbDVUCSJm3EspkuAVZp5j1ckkvqC4Gr_aR5_VhDvozK5WDExtjTKpRkE$>:
Some doubts here:
Balance Details - CommerceHub | Developer Studio<https://developer.fiserv.com/product/CommerceHub/docs/?path=docs/Resources/Master-Data/Balance-Components.md&branch=main#tab-balance>
This is the content page.
Doubt #1:
The table says
beginngingBalance
number
18,3
Credit card number
Beginning balance description should be something like: Opening balance.
blanceType
string
17
Indicates type<https://developer.fiserv.com/product/CommerceHub/docs?path=docs/Resources/Master-Data/Balance-Components.md#balance-type> of account balance represents.
Balance type can be either Credit or Debit. Probably within the drawing limit or some more explanation according to account type.
The description as balanceType Credit (positive) or Debit (negative) balance.
Can we reach out to a SME for changing these two descriptions?

Doubt #2:
Technically in this topic, there is no section as balance-type
We cannot set anchor to the object `blanceType`using <a herf> .

Can I therefore suggest that we can remove the anchor link for the word [the type].  It is a direct explanation for the column.
Please confirm/ suggest.
docs/Resources/Master-Data/Card-Details.md<https://urldefense.com/v3/__https://github.com/Fiserv/Commerce-Hub/pull/1386*discussion_r1829547814__;Iw!!P9vvK-4S!mlbvk1v5p2tGEMLgISVw-KEpOvsbc03VA_R_lCu7XkG_SkLZbDVUCSJm3EspkuAVZp5j1ckkvqC4Gr_aR5_VhDvozK5WDExtbEd0r7Y$>: Done

 docs/Resources/Master-Data/Debit-Pinless-Indicator.md<https://urldefense.com/v3/__https://github.com/Fiserv/Commerce-Hub/pull/1386*discussion_r1829563461__;Iw!!P9vvK-4S!mlbvk1v5p2tGEMLgISVw-KEpOvsbc03VA_R_lCu7XkG_SkLZbDVUCSJm3EspkuAVZp5j1ckkvqC4Gr_aR5_VhDvozK5WDExtO7EMsNY$>:
done for all except for object/field reference: `debitPinlessIndicator` and tags. Please confirm.

 docs/Resources/Master-Data/Merchant-Category-Code.md<https://urldefense.com/v3/__https://github.com/Fiserv/Commerce-Hub/pull/1386*discussion_r1829565256__;Iw!!P9vvK-4S!mlbvk1v5p2tGEMLgISVw-KEpOvsbc03VA_R_lCu7XkG_SkLZbDVUCSJm3EspkuAVZp5j1ckkvqC4Gr_aR5_VhDvozK5WDExt76Oj72Y$>:
 Made : bold  and removed extra line.

docs/Resources/Master-Data/Payment-Receipt.md<https://urldefense.com/v3/__https://github.com/Fiserv/Commerce-Hub/pull/1386*discussion_r1829569363__;Iw!!P9vvK-4S!mlbvk1v5p2tGEMLgISVw-KEpOvsbc03VA_R_lCu7XkG_SkLZbDVUCSJm3EspkuAVZp5j1ckkvqC4Gr_aR5_VhDvozK5WDExtmWuenks$>:
Done

 Regards,
Srividya

________________________________

**Comment by svFiserv** [comment]
Hi William,

Have made the suggested changes.

docs/Archive/Payment-JS/JSV1-Request.md<https://urldefense.com/v3/__https://github.com/Fiserv/Commerce-Hub/pull/1386*discussion_r1829438760__;Iw!!P9vvK-4S!mlbvk1v5p2tGEMLgISVw-KEpOvsbc03VA_R_lCu7XkG_SkLZbDVUCSJm3EspkuAVZp5j1ckkvqC4Gr_aR5_VhDvozK5WDExtseTHduc$>:
Done

docs/Resources/Master-Data/Balance-Components.md<https://urldefense.com/v3/__https://github.com/Fiserv/Commerce-Hub/pull/1386*discussion_r1829545098__;Iw!!P9vvK-4S!mlbvk1v5p2tGEMLgISVw-KEpOvsbc03VA_R_lCu7XkG_SkLZbDVUCSJm3EspkuAVZp5j1ckkvqC4Gr_aR5_VhDvozK5WDExtjTKpRkE$>:
Some doubts here:
Balance Details - CommerceHub | Developer Studio<https://developer.fiserv.com/product/CommerceHub/docs/?path=docs/Resources/Master-Data/Balance-Components.md&branch=main#tab-balance>
This is the content page.
Doubt #1:
The table says
beginngingBalance
number
18,3
Credit card number
Beginning balance description should be something like: Opening balance or balance at the start of the day.
blanceType
string
17
Indicates type<https://developer.fiserv.com/product/CommerceHub/docs?path=docs/Resources/Master-Data/Balance-Components.md#balance-type> of account balance represents.
There is no clarity on balance type other than what we see on this row of the table.
Can we have a short section about Balance Type?
In copilot search, come across:

  1.
Available Balance

  *   This is the amount of money you can use or withdraw from your account at any given time. It includes all cleared deposits and subtracts any pending transactions.

2. Current Balance

  *   Also known as the ledger balance, this is the total amount of money in your account at the end of the previous business day. It includes all transactions that have been processed but not those that are still pending.

3. Ledger Balance

  *   Similar to the current balance, this is the balance at the start of the business day, reflecting all transactions that have been posted up to that point.

4. Collected Balance

  *   This is the portion of your balance that has been cleared and is available for withdrawal. It excludes any deposits that are still pending clearance.

5. Projected Balance

  *   This balance includes all pending transactions and gives an estimate of what your balance will be once all transactions are processed.

6. Book Balance

  *   This is the balance according to your own records, which may include transactions that have not yet been processed by the bank.

7. Float Balance

  *   This refers to the amount of money that is in the process of being transferred between accounts but has not yet been cleared.

Do we reach out to SME to learn whether they are applicable to our context?
Doubt/suggestion  #2:
Technically in this topic, there is no section as balance-type
We cannot set anchor to the object `blanceType`using <a herf> .

Can I therefore suggest that we can remove the anchor link for the word [the type].  It is a direct explanation for the column.
Please confirm/ suggest.
docs/Resources/Master-Data/Card-Details.md<https://urldefense.com/v3/__https://github.com/Fiserv/Commerce-Hub/pull/1386*discussion_r1829547814__;Iw!!P9vvK-4S!mlbvk1v5p2tGEMLgISVw-KEpOvsbc03VA_R_lCu7XkG_SkLZbDVUCSJm3EspkuAVZp5j1ckkvqC4Gr_aR5_VhDvozK5WDExtbEd0r7Y$>: Done

 docs/Resources/Master-Data/Debit-Pinless-Indicator.md<https://urldefense.com/v3/__https://github.com/Fiserv/Commerce-Hub/pull/1386*discussion_r1829563461__;Iw!!P9vvK-4S!mlbvk1v5p2tGEMLgISVw-KEpOvsbc03VA_R_lCu7XkG_SkLZbDVUCSJm3EspkuAVZp5j1ckkvqC4Gr_aR5_VhDvozK5WDExtO7EMsNY$>:
done for all except for object/field reference: `debitPinlessIndicator` and tags. Please confirm.

 docs/Resources/Master-Data/Merchant-Category-Code.md<https://urldefense.com/v3/__https://github.com/Fiserv/Commerce-Hub/pull/1386*discussion_r1829565256__;Iw!!P9vvK-4S!mlbvk1v5p2tGEMLgISVw-KEpOvsbc03VA_R_lCu7XkG_SkLZbDVUCSJm3EspkuAVZp5j1ckkvqC4Gr_aR5_VhDvozK5WDExt76Oj72Y$>:
 Made : bold  and removed extra line.

docs/Resources/Master-Data/Payment-Receipt.md<https://urldefense.com/v3/__https://github.com/Fiserv/Commerce-Hub/pull/1386*discussion_r1829569363__;Iw!!P9vvK-4S!mlbvk1v5p2tGEMLgISVw-KEpOvsbc03VA_R_lCu7XkG_SkLZbDVUCSJm3EspkuAVZp5j1ckkvqC4Gr_aR5_VhDvozK5WDExtmWuenks$>:
Done

 Regards,
Srividya

________________________________

**Comment by wwfiserv** [comment]
@srija-fiserv for Balance inquiry remove the link, there should be a table that defines what the balance type is, will have to open an inquiry ticket if I can't find the original Solution and documentation request. I'm guessing the prior tech writer never got the info before publishing.

**Pull request merged**